src/pip/_internal/operations/install/wheel.py

```diff
@@ -387,6 +385,7 @@ def clobber(
                             ) + ', '.join(info_dir)
                         )
                         info_dir.append(destsubdir)
+                subdirs[:] = [s for s in subdirs if not s.endswith('.data')]
```

Comment:
In 2fd0f8d35a9, a comment would have helped me understand the reason for this line that seems useless since `subdirs` isn't used in the rest of the function.

For other readers, cf https://docs.python.org/3/library/os.html#os.walk:
```
When topdown is True, the caller can modify the dirnames list in-place (perhaps using del or
slice assignment), and walk() will only recurse into the subdirectories whose names remain in
dirnames; this can be used to prune the search, impose a specific order of visiting, or even to
inform walk() about directories the caller creates or renames before it resumes walk() again.
```
At least I learned something :grin:

Reply:
Good feedback, thank you.

This line is not long for the world anyway, since we will not be walking over directories but iterating over names in the zip file.